Form-Controls/styles.css

```diff
@@ -0,0 +1,74 @@
+
+body{
+    background: #F68E5F;  
+    text-align: center;
+    width: 30%;
```

Comment:
Do you always want this element to be 30% width? 30% of what? What would happen if you zoomed this layout to 200%? (Try it!)

Look at this layout in DevTools using the Device mode and have a think https://developer.chrome.com/docs/devtools/device-mode/